VersionControlUI/src/DiffManager.h

```diff
@@ -91,6 +92,11 @@ class VERSIONCONTROLUI_API DiffManager
 		 */
 		Model::TreeManager* createTreeManagerFromVersion(FilePersistence::GitRepository& repository, QString version);
 
+		/**
+		 * Returns all items which have any of their parents present in \a items.
+		 */
+		static QSet<Visualization::Item*> findAllItemsWithParentPresent(QSet<Visualization::Item*> items);
```

Comment:
A slightly better name is `findAllItemsWithParentIn`

Are you talking about direct parents or Ancestors? If the second then please use the appropriate word.